README.md

```diff
@@ -51,13 +51,35 @@ Usage
 
     $ python3 iq-apply-conf.py --help
 
+Docker Usage
+
+    Build the image by your own:
+
+        $ docker build -t iq-server-casc:latest .
+
+    Build the image by your own with a custom base image:
+
+        $ docker build -t iq-server-casc:latest --build-arg BASEIMAGE=python:3.9.6-alpine3.14 .
+
+    In case you need some custom user ids or group ids, you could just build the image with those args:
+
+        $ docker build -t iq-server-casc:latest --build-arg USER_ID=1001 --build-arg GROUP_ID=1001 .
+
+    Run with local image:
+
+        docker run --rm -i -v $PWD:/data iq-server-casc:latest <iq-script>
+
+    Run with the latest upstream image:
+
+        docker run --rm -i -v $PWD:/data kehrhardt/iq-server-casc:latest <iq-script>
+
 
 Usage: iq-apply-config [ARGS]...
 
   Example usage:
 
     # Run python script though docker container with all packages installed on it!
-    docker run -w /tmp --rm -i -v $PWD:/tmp broadinstitute/python-requests /tmp/iq-apply-conf.py -f /tmp/conf/<conf-file>.json -u http://<iq-hostname>:<iq-port> -a <user>:<password>
+    docker run --rm -i -v $PWD:/data kehrhardt/iq-server-casc:latest iq-apply-conf -f /data/conf/<conf-file>.json -u http://<iq-hostname>:<iq-port> -a <user>:<password>
```

Comment:
Add comment to denote /data/conf/<config-file>.json is a unit test file.
Use /data/<config-file>.json to apply 'scraped' data.